<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkQBallReconstructionViewControls</class>
 <widget class="QWidget" name="QmitkQBallReconstructionViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>844</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>QmitkQBallReconstructionViewControls</string>
  </property>
  <property name="styleSheet">
   <string>QCommandLinkButton:disabled {
  border: none;
}

QGroupBox {
  background-color: transparent;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="verticalSpacing">
    <number>25</number>
   </property>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QComboBox" name="m_QBallReconstructionMethodComboBox">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>Numerical</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Standard</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Solid Angle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ADC-Profile only</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Raw Signal only</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Multi-Shell</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="m_Description">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QFrame" name="m_ShFrame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="4" column="0">
             <widget class="QLabel" name="m_QBallReconstructionLambdaTextLabel_2">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="toolTip">
               <string>Regularization Parameter</string>
              </property>
              <property name="text">
               <string>Lambda:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QDoubleSpinBox" name="m_QBallReconstructionLambdaLineEdit">
              <property name="toolTip">
               <string>Regularization factor</string>
              </property>
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.001000000000000</double>
              </property>
              <property name="value">
               <double>0.006000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="m_QBallReconstructionMaxLLevelTextLabel_2">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>SH-Order:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="m_QBallReconstructionMaxLLevelComboBox">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="currentIndex">
               <number>-1</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="m_QBallReconstructionThresholdLabel_2">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>B0 Threshold</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="m_QBallReconstructionThreasholdEdit">
              <property name="maximum">
               <number>10000</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="m_OutputOdfImage">
           <property name="text">
            <string>Output Sampled ODF Image</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCommandLinkButton" name="m_ButtonStandard">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Start Reconstruction</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="m_InputData">
     <property name="title">
      <string>Input Data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="toolTip">
         <string>Input for Q-Ball reconstruction.</string>
        </property>
        <property name="text">
         <string>Raw DWI:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QmitkDataStorageComboBox" name="m_ImageBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="m_QBallSelectionBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Multi-Shell Reconstruction</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Convert SH to sampled ODF image</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="3" column="0">
       <widget class="QCommandLinkButton" name="m_ConvertButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Convert</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="toolTip">
            <string>Input for Q-Ball reconstruction.</string>
           </property>
           <property name="text">
            <string>SH Image:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QmitkDataStorageComboBox" name="m_ShImageBox"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">QmitkDataStorageComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
